crates/gpui/src/platform/windows/window.rs

```diff
@@ -1709,3 +1800,21 @@ fn logical_point(x: f32, y: f32, scale_factor: f32) -> Point<Pixels> {
 
 // https://learn.microsoft.com/en-us/windows/win32/api/shellapi/nf-shellapi-dragqueryfilew
 const DRAGDROP_GET_FILES_COUNT: u32 = 0xFFFFFFFF;
+
+enum FullScreenState {
+    None,
+    Normal {
+        style: WINDOW_STYLE,
+        x: i32,
+        y: i32,
+        cx: i32,
+        cy: i32,
+    },
+    Maximized(WINDOW_STYLE),
```

Comment:
Why is maximized included here?

Reply:
I thought it was necessary to separate the cases of going full screen from the normal window and the case of  the maximized one, but I found that the same process was fine, so I removed it.